extension/controller/translation/Translation.js

```diff
@@ -149,6 +150,8 @@ class Translation {
         this.translationsMessagesCounter += 1;
         translationMessage.messageID = this.translationsMessagesCounter;
         translationMessage.sourceParagraph = sourceParagraph;
+        // let's revisit this later, since passing false here when there's plain text is breaking
+        translationMessage.isHTML = true; // this.htmlRegex.test(sourceParagraph);
```

Comment:
As proposed by @andrenatal, we will figure out this part together and in a separate issue to unblock QA.